src/VisualStudio/CSharp/Impl/LanguageService/CSharpCodeCleanupFixer.cs

```diff
@@ -372,7 +374,8 @@ async Task<Solution> ApplyFixAsync(ProgressTracker progressTracker, Cancellation
         private Task<bool> FixTextBufferAsync(TextBufferCodeCleanUpScope textBufferScope, ICodeCleanUpExecutionContext context, CancellationToken cancellationToken)
         {
             var buffer = textBufferScope.SubjectBuffer;
-            return FixAsync(buffer.GetWorkspace(), ApplyFixAsync, context, cancellationToken);
+            var documentName = buffer.CurrentSnapshot.GetOpenDocumentInCurrentContextWithChanges().Name;
```

Comment:
this could return null (http://source.roslyn.io/#Microsoft.CodeAnalysis.Workspaces/Workspace/TextExtensions.cs,47)?
 or are we sure in this code path, it can't be?